workers/unity/Packages/com.improbable.gdk.core/Tests/Editmode/GameObjectRepresentation/EntityGameObjectLinkerTests.cs

```diff
@@ -23,6 +24,7 @@ public void Setup()
         {
             world = new World("TestWorld");
             entityManager = world.GetOrCreateManager<EntityManager>();
+            world.CreateManager<WorkerSystem>(null, null, "TestWorker", Vector3.zero);
```

Comment:
do we need that line?

Reply:
Unfortunately yes. Upon creating the `EntityGameObjectLinker`, that object will getorcreate a `GameObjectDispatcherSystem` which will attempt to call `World.GetExistingManager<WorkerSystem>()`